Tools/AP_Periph/AP_Periph.cpp

```diff
@@ -142,6 +142,10 @@ void AP_Periph_FW::init()
         printf("Reboot after watchdog reset\n");
     }
 
+#ifdef NODE_STATS_ENABLED
```

Comment:
Please use `#if AP_STATS_ENABLED` here - so different namespacing, and checking for value rather than definition.

This will require a change to chibios_hwdef.py where we write out the periph; we do a lot of that... https://github.com/ardupilot/ardupilot/blob/master/libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py#L2761

Reply:
Yes, we can use AP_STATS_ENABLE with no issues.
Mostly, we are interested in the number of times the peripheral board has reset. To keep a track of unwanted reset in the system due to noise or watchdog.